infrastructure/stacks/capacity_management/lambda.tf

```diff
@@ -0,0 +1,319 @@
+/**************************
+//
+//ORGANISATIONS DATA MANAGER LAMBDA
+//
+**************************/
+
+
+module "organisations-data-manager-lambda" {
+  source = "../../modules/lambda"
+
+  function_name = "organisations-data-manager"
+  description   = "Microservice for interacting with organisations dynamodb table"
+
+  policy_jsons = [
+    <<-EOT
+        {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Sid": "DynamodbTable",
+                    "Effect": "Allow",
+                    "Action": [
+                        "dynamodb:PutItem",
+                        "dynamodb:DeleteItem",
+                        "dynamodb:GetItem",
+                        "dynamodb:Scan",
+                        "dynamodb:Query",
+                        "dynamodb:UpdateItem"
+                    ],
+                    "Resource": [
+                        "${module.dynamodb_organisations_table.dynamodb_table_arn}"
+                    ]
+                }
+            ]
+        }
+        EOT
+  ]
+}
+
+#    allowed_triggers = {
+#        AllowExecutionFromAPIGateway = {
+#        service    = "apigateway"
+#        source_arn = "${aws_api_gateway_rest_api.DoS_REST.execution_arn}/*/*"
+#        }
+#    }
+#    }
+
+#    module "live-alias-organisations-data-manager" {
```

Comment:
Dead code we can remove ?

Reply:
Needed as part of the work coming up in the next sprint